__tests__/setup-go.test.ts

```diff
@@ -52,18 +54,22 @@ describe('setup-go', () => {
     dlSpy = jest.spyOn(tc, 'downloadTool');
     exSpy = jest.spyOn(tc, 'extractTar');
     cacheSpy = jest.spyOn(tc, 'cacheDir');
-    getSpy = jest.spyOn(im, 'getVersions');
+    getSpy = jest.spyOn(im, 'getVersionsDist');
+    getManifestSpy = jest.spyOn(tc, 'getManifestFromRepo');
 
     // io
     whichSpy = jest.spyOn(io, 'which');
     existsSpy = jest.spyOn(fs, 'existsSync');
     mkdirpSpy = jest.spyOn(io, 'mkdirP');
 
+    // gets
+    getManifestSpy.mockImplementation(() => <tc.IToolRelease[]>goTestManifest);
+
     // writes
     cnSpy = jest.spyOn(process.stdout, 'write');
     logSpy = jest.spyOn(console, 'log');
-    dbgSpy = jest.spyOn(main, '_debug');
-    getSpy.mockImplementation(() => <im.IGoVersion[]>goJsonData);
+    dbgSpy = jest.spyOn(core, 'debug');
```

Comment:
Did this actually mock out the debug output or were you seeing it in the output?  I was having an issue with that.  If so, then great- this is cleaner

Reply:
We want to clarify one point, as I understand that we should not receive output while these lines are commented out?
```
    dbgSpy.mockImplementation(msg => {
      // uncomment to see debug output
      // process.stderr.write(msg + '\n');
    });
```
If we uncomment them, should we get a conclusion or not? On normal startup, we do not get debug output.